Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -1606,9 +1825,18 @@ def threat_to_incident(threat):
         disconnect_agent_from_network()
     elif demisto.command() == 'sentinelone-broadcast-message':
         broadcast_message()
-
+    elif demisto.command() == 'sentinelone-get-events':
+        get_events()
+    elif demisto.command() == 'sentinelone-create-query':
+        create_query()
+    elif demisto.command() == 'sentinelone-get-processes':
+        get_processes()
+    elif demisto.command() == 'sentinelone-shutdown-agent':
+        shutdown_agents()
+    elif demisto.command() == 'sentinelone-uninstall-agent':
+        uninstall_agent()
 
 except Exception as e:
     LOG(str(e))
     LOG.print_log()
-    raise
```

Comment:
we should add a condition here in case we're in `fetch-incidents`, then we would want to raise the error and not use `return_error`